src/OpenTok/Util/Client.php

```diff
@@ -124,13 +135,22 @@ public function isConfigured()
 
     private function createAuthHeader()
     {
+        if (!is_null($this->applicationId) && !is_null($this->privateKeyPath)) {
```

Comment:
As above, I would just check one of the option flags instead, but otherwise this is the idea